Request that bug reports be sent to us rather than upstream.
--- mkpasswd.c.orig	2023-10-08 15:58:33.000000000 -0500
+++ mkpasswd.c	2023-10-22 20:40:51.000000000 -0500
@@ -535,7 +535,7 @@
 "If no SALT is specified, a random one is generated.\n"
 "If TYPE is 'help', available methods are printed.\n"
 "\n"
-"Report bugs to %s.\n"), "<md+whois@linux.it>");
+"Report bugs to %s.\n"), "<https://trac.macports.org/newticket>");
     exit(error);
 }
 
--- whois.c.orig	2023-07-22 10:56:56.000000000 -0500
+++ whois.c	2023-07-24 15:41:53.000000000 -0500
@@ -240,7 +240,7 @@
 	    break;
 	case 1:
 	    fprintf(stdout, _("Version %s.\n\nReport bugs to %s.\n"),
-		    VERSION, "<md+whois@linux.it>");
+		    VERSION, "<https://trac.macports.org/newticket>");
 	    exit(EXIT_SUCCESS);
 	default:
 	    usage(EXIT_FAILURE);
